demo_exam/CheckDate.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+int getLen(char* arr)
+{
+	int index = 0;
+	if (arr[0] == '\0')
+	{
+		return 0;
+	}
+	while (arr[index] != 0)
+	{
+		index++;
+	}
+	return index + 1;
+}
+
+bool checkDate(char* array)
+{
+	int index = 0;
+	int len = getLen(array);
+	if (len != 5)
```

Comment:
You could have that as a constant (same for boundaries and other 'magic' literals below.